templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+zk:
+  tty: true
+  image: rawmind/rancher-zk:0.0.1-1
```

Comment:
Could this be linked to the Rancher Catalog zookeeper instead of launching its own?